microservices/optimizer/utils/db_manager.py

```diff
@@ -1 +1,48 @@
 """Utility module for managing optDB"""
+
+from datetime import datetime
+from typing import Any
+from pymongo import MongoClient
+
+# Number of execution times to keep track of for each test class
+NUM_ENTRIES: int = 20
+
+
+# pylint: disable=too-few-public-methods
+class DBManager:
+    """Class for managing optDB"""
+
+    def __init__(
+        self, uri: str = "mongodb://localhost:27017/", db_name: str = "optDB"
```

Comment:
Due to Docker Compose networking, the IP is something different:
```suggestion
        self, uri: str = "mongodb://optdb:27017/", db_name: str = "optDB"
```